pep-0634.rst

```diff
@@ -177,8 +177,8 @@ Patterns
 The top-level syntax for patterns is as follows::
 
   patterns: open_sequence_pattern | pattern
-  pattern: walrus_pattern | or_pattern
-  walrus_pattern: capture_pattern ':=' or_pattern
+  pattern: as_pattern | or_pattern
+  as_pattern: or_pattern 'as' capture_pattern
```

Comment:
I forget, do we still need/want to unify the grammars in the implementation and spec? As it currently stands, `as_pattern` isn't a rule, it's just folded into `pattern`:

```
pattern[expr_ty]:
    | value=or_pattern 'as' target=NAME { _Py_MatchAs(value, target->v.Name.id, EXTRA) }
    | or_pattern
```

Reply:
(this isn't the only difference, though)

Reply:
I'd like the implementation eventually to use the same grammar as the PEP. I'd rather not change the PEP (unless we find bugs) because I use the rule names (e.g. "as pattern") in the text without explanation.